src/main/java/org/owasp/wrongsecrets/oauth/TokenController.java

```diff
@@ -0,0 +1,59 @@
+package org.owasp.wrongsecrets.oauth;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.UUID;
+
+@Slf4j
+@Controller
+public class TokenController {
+
+    private final String dockerMountPath;
+
+    public TokenController(@Value("${challengedockermtpath}") String dockerMountPath) {
```

Comment:
i believe this is used for something else already, can you use a new variable please :)?

Reply:
Additional secret key was added.